### DISCUSSION TITLE
[Windows][Files → Delete]: inherit Recycle Bin's confirmation settings

### DISCUSSION THREAD

**Discussion by PetbkA** [doublecmd/doublecmd#1206] [Suggestions]
> Double Commander
> Version: **1.1.0 beta**
> Revision: 32
> Commit: **cd3a69c65**
> Build date: 2023/08/05
> Lazarus: 2.2.7.0
> FPC: 3.2.2
> Platform: **x86_64-Win64-win32/win64**
> OS version: **Windows 10 Home 22H2 x86_64**

_Recycle Bin_ in Windows has option [_Display delete confirmation dialog_](https://www.makeuseof.com/windows-disable-delete-confirmation/), but DC's _Delete_ / _cm_Delete_ command doesn't inherit this value and deletes _without_ confirmation by default, unless you set _confirmation_=`1` in command parameters.

I suggest to inherit _Recycle Bin_'s _Display delete confirmation dialog_ value, if no parameters are specified for _cm_Delete_ command

**Comment by alexx2000** [comment]
DC has its own settings. By default `cm_Delete` show confirmation dialog, if not then you disabled it in the [options](https://doublecmd.github.io/doc/en/configuration.html#ConfigOperations).

**Comment by PetbkA** [comment]
Indeed, didn't saw flag for _Delete to **trash** operation_
